### PR TITLE
Add project description

### PR DESCRIPTION
**Pull request by ZijiaZhang**
Hi team,
 I added the project description to README. Please take a look.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,52 @@
...
+- Backend management system (Administrator)
+- Able to set blog post to public/private/to friends
+- Announcement system (public posts)
+- Comment, Like to a post
+- BlackList
+## 2-3 stretch requirements (plan to complete 1!)
+- Group chat
+- Apps to help users get to know each other 
+
+# Designs
```
Comment (xddenise) [inline README.md:49]: We might want to finalize the design in the group chat before merging

File: README.md [hunk 2]
```diff
@@ -2,3 +2,52 @@
 
 [![Build Status](https://travis-ci.com/ZijiaZhang/CPSC436_Project.svg?branch=master)](https://travis-ci.com/ZijiaZhang/CPSC436_Project)
 [![codecov](https://codecov.io/gh/ZijiaZhang/CPSC436_Project/branch/master/graph/badge.svg)](https://codecov.io/gh/ZijiaZhang/CPSC436_Project)
+
+## Project Description: 
+This project aims to provide more precise people finder and chat environment. 
+It will allows users to make new friends easily. They will be able to find pareners in the same lectures,
```
Comment (xddenise) [inline README.md:8]: Typo: pareners -> partners

**Comment by NingyuanXu** [comment]
Hi Team, 

I update the attached image into the new ones. please check

Will

File: README.md [hunk 3]
```diff
@@ -2,3 +2,50 @@
 
 [![Build Status](https://travis-ci.com/ZijiaZhang/CPSC436_Project.svg?branch=master)](https://travis-ci.com/ZijiaZhang/CPSC436_Project)
 [![codecov](https://codecov.io/gh/ZijiaZhang/CPSC436_Project/branch/master/graph/badge.svg)](https://codecov.io/gh/ZijiaZhang/CPSC436_Project)
+
+## Project Description: 
+This project aims to provide more precise people finder and chat environment. 
+It will allows users to make new friends easily. They will be able to find partners in the same lectures, 
+labs and also shout out to others to form a study group to prepare for their exams.
```
Comment (xddenise) [inline README.md:9]: Our project aims to provide a friendly and focused environment for UBC students who are looking for friends, activities, or simply want to stay updated with what’s going on. Users can do things such as finding people who are in the same lecture or lab section as them, forming a study group for midterms or finals, looking for carpooling, etc.

File: README.md [hunk 4]
```diff
@@ -2,3 +2,50 @@
...
+
+## Project Description: 
+This project aims to provide more precise people finder and chat environment. 
+It will allows users to make new friends easily. They will be able to find partners in the same lectures, 
+labs and also shout out to others to form a study group to prepare for their exams. 
+
+## Who is it for?
+UBC students who want to find friends with same classes, hobbies, and etc
+## What will it do? 
+- Blog website to share texts and pictures, like Twitter
```
Comment (xddenise) [inline README.md:14]: - News feed page to share texts and pictures, like Twitter

File: README.md [hunk 5]
```diff
@@ -2,3 +2,50 @@
...
+- Chat room
+- Add friends
+## What type of data will it store?
+- Username/ Password
+- Basic User information
+- Chat history
+- Message history in Blog
```
Comment (xddenise) [inline README.md:21]: - activity history (number of likes or comments) of posts in the news feed page

File: README.md [hunk 6]
```diff
@@ -2,3 +2,50 @@
...
+## What will users be able to do with this data?
+- Log in/off
+- Search for, query by hobbies, gender, classes etc (search option)
+- Find/Add friends with similar hobby tags
+- Interact on blog posts, reply, Like
```
Comment (xddenise) [inline README.md:26]: - Interact on posts, such as comment or like

File: README.md [hunk 7]
```diff
@@ -2,3 +2,50 @@
...
+## What is some additional functionality you can add/remove based on time constraints?
+- Apps to help users get to know each other
```
Comment (xddenise) [inline README.md:28]: Game apps?

File: README.md [hunk 8]
```diff
@@ -2,3 +2,50 @@
...
+- Chat history
+- Message history in Blog
+## What will users be able to do with this data?
+- Log in/off
+- Search for, query by hobbies, gender, classes etc (search option)
+- Find/Add friends with similar hobby tags
+- Interact on blog posts, reply, Like
+## What is some additional functionality you can add/remove based on time constraints?
+- Apps to help users get to know each other   
+- Better Recommendation on blog feeds
```
Comment (xddenise) [inline README.md:29]: blog feeds -> news feed

File: README.md [hunk 9]
```diff
@@ -2,3 +2,50 @@
...
+ 
+# Project task requirements:
+## 3-5 minimal requirements (will definitely complete)
+- User Logon/off
+- Blog to post, share thought in text, picture
```
Comment (xddenise) [inline README.md:34]: Blog -> news feed page

File: README.md [hunk 10]
```diff
@@ -2,3 +2,50 @@
...
+## 3-7 "standard" requirements (will most likely complete)
+- Recommend by tags (people you may be interested in)
+- Backend management system (Administrator)
+- Able to set blog post to public/private/to friends
+- Announcement system (public posts)
+- Comment, Like to a post
+- BlackList
+## 2-3 stretch requirements (plan to complete 1!)
+- Group chat
+- Apps to help users get to know each other
```
Comment (xddenise) [inline README.md:47]: Game apps?

**Pull request merged**